points/SO3/LICENSE

```diff
@@ -0,0 +1,11 @@
+The SO3 points clouds have been generated from the ISOI software available
+at this webpage : 
+https://mitchell-web.ornl.gov/SOI/index.php
```

Comment:
This webpage mentions 2 software, SOI (copyrighted but free for non-commercial use) and ISOI (GPL). If it is indeed the second one that was used, it would be worth mentioning its license here.

Reply:
It is the GPL one. It is clearer after removing the 2 papers information from the LICENSE file on [6648f2d](https://github.com/GUDHI/gudhi-data/pull/11/commits/6648f2d633ff8823066161de005074b9672bbee3)